[package] <qt>/<6.8.3>: missing system lib when building with MSVC

### Description

When building an application that requires Qt 6.8.3 static, link fails due to symbol not found.

QWindowsIntegrationPlugin needs uiautomationcore since 6.8.0, see:
https://github.com/qt/qtbase/blob/6.8.3/src/plugins/platforms/windows/CMakeLists.txt#L204
https://github.com/qt/qtbase/blob/6.8.0/src/plugins/platforms/windows/CMakeLists.txt#L202

Would need something like
```
 if Version(self.version) >= "6.8.0":
                    self.cpp_info.components["qtQWindowsIntegrationPlugin"].system_libs += ["uiautomationcore"]
```
just after https://github.com/conan-io/conan-center-index/blob/master/recipes/qt/6.x.x/conanfile.py#L1188
or something like that.

Work around: add uiautomationcore to target_link_libraries() which is not ideal.


### Package and Environment Details

* Package Name/Version: **qt/6.8.3**
* Operating System+version: **Windows 11**
* Compiler+version: **MSVC 2022**
* Conan version: **conan 2.13.0**
* Python version: **Python 3.8.19**

### Conan profile

Host profile:
[settings]
arch=x86_64
build_type=Debug
compiler=msvc
compiler.cppstd=17
compiler.runtime=dynamic
compiler.runtime_type=Debug
compiler.version=194
os=Windows
ffmpeg/*:build_type=Release
[conf]
tools.cmake.cmaketoolchain:generator=Ninja
tools.microsoft.bash:active=False
tools.microsoft.bash:subsystem=msys2

Build profile:
[settings]
arch=x86_64
build_type=Debug
compiler=msvc
compiler.cppstd=17
compiler.runtime=dynamic
compiler.runtime_type=Debug
compiler.version=194
os=Windows
ffmpeg/*:build_type=Release
[conf]
tools.cmake.cmaketoolchain:generator=Ninja
tools.microsoft.bash:active=False
tools.microsoft.bash:subsystem=msys2

### Steps to reproduce

Patch conan-center-index so that test package load QWindowsIntegrationPlugin.

```Patch
diff --git a/recipes/qt/6.x.x/test_package/CMakeLists.txt b/recipes/qt/6.x.x/test_package/CMakeLists.txt
index 45e4bb9ec9..a6b60fc5c3 100644
--- a/recipes/qt/6.x.x/test_package/CMakeLists.txt
+++ b/recipes/qt/6.x.x/test_package/CMakeLists.txt
@@ -8,6 +8,10 @@ target_link_libraries(${PROJECT_NAME} PRIVATE Qt6::Core Qt6::Network Qt6::Sql Qt
 target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_17)
 set_target_properties(${PROJECT_NAME} PROPERTIES AUTOMOC ON AUTORCC ON)

+if (WIN32)
+    target_link_libraries(${PROJECT_NAME} PRIVATE Qt6::QWindowsIntegrationPlugin)
+endif()
+
 # Only running this Qt macros in macOS
 if (APPLE)
     # Related to https://github.com/conan-io/conan-center-index/issues/20574
diff --git a/recipes/qt/6.x.x/test_package/test_package.cpp b/recipes/qt/6.x.x/test_package/test_package.cpp
index 5374ecf402..acaee07b3f 100644
--- a/recipes/qt/6.x.x/test_package/test_package.cpp
+++ b/recipes/qt/6.x.x/test_package/test_package.cpp
@@ -11,6 +11,9 @@

 #include "greeter.h"

+#include <QtPlugin>
+Q_IMPORT_PLUGIN(QWindowsIntegrationPlugin)
+
 int main(int argc, char *argv[]){
     QCoreApplication app(argc, argv);
     QCoreApplication::setApplicationName("Application Example");
```

Then build (options shouldn't mater, it's what I had at hand)

conan test .\recipes\qt\6.x.x\test_package\ qt/6.8.3 -o="qt/*:qtmultimedia=True" -o="qt/*:qtsvg=True" -o="qt/*:qttranslations=True" -o="qt/*:with_openal=False" -nr

Change line for:
`target_link_libraries(${PROJECT_NAME} PRIVATE Qt6::QWindowsIntegrationPlugin uiautomationcore)` 
and it will link now.

### Logs

<details><summary>Click to expand log</summary>

```
(conan2) PS C:\Users\usernameredacted\Projets\conan-center-index> conan test .\recipes\qt\6.x.x\test_package\ qt/6.8.3 -o="qt/*:qtmultimedia=True" -o="qt/*:qtsvg=True" -o="qt/*:qttranslations=True" -o="qt/*:with_openal=False" -nr
detect_api: Found msvc 17
detect_api: Found msvc 17

======== Input profiles ========
Profile host:
[settings]
arch=x86_64
build_type=Debug
compiler=msvc
compiler.cppstd=17
compiler.runtime=dynamic
compiler.runtime_type=Debug
compiler.version=194
os=Windows
ffmpeg/*:build_type=Release
[options]
qt/*:qtmultimedia=True
qt/*:qtsvg=True
qt/*:qttranslations=True
qt/*:with_openal=False
[conf]
tools.cmake.cmaketoolchain:generator=Ninja
tools.microsoft.bash:active=False
tools.microsoft.bash:subsystem=msys2

Profile build:
[settings]
arch=x86_64
build_type=Debug
compiler=msvc
compiler.cppstd=17
compiler.runtime=dynamic
compiler.runtime_type=Debug
compiler.version=194
os=Windows
ffmpeg/*:build_type=Release
[conf]
tools.cmake.cmaketoolchain:generator=Ninja
tools.microsoft.bash:active=False
tools.microsoft.bash:subsystem=msys2


======== Launching test_package ========

======== Computing dependency graph ========
Graph root
    qt/6.8.3 (test package): C:\Users\usernameredacted\Projets\conan-center-index\recipes\qt\6.x.x\test_package\conanfile.py
Requirements
    brotli/1.1.0#d56d7bb9ca722942aba17369cb5c0519 - Cache
    bzip2/1.0.8#d00dac990f08d991998d624be81a9526 - Cache
    double-conversion/3.3.0#33321c201741cc32b51169c6d2d05e60 - Cache
    freetype/2.13.2#5d2563803c8558d4ef47271a82c73d20 - Cache
    glib/2.78.3#a15f775789e881a2bc3c9693aa69b2c6 - Cache
    harfbuzz/8.3.0#c83405ed297bad6c9e9587e2a19c686a - Cache
    libffi/3.4.4#1520e366dc9a6a38898b4993bb198748 - Cache
    libgettext/0.22#35d2811b2dd27a98f69e4daa86ca2000 - Cache
    libiconv/1.17#1ae2f60ab5d08de1643a22a81b360c59 - Cache
    libpng/1.6.47#d7ad969de6b0a72704cf03d6ef17648b - Cache
    libpq/15.4#cca9970f349bcdb6c8b35acd017ee5aa - Cache
    md4c/0.4.8#d7347fea4b215dfc7041f88a539508d0 - Cache
    openssl/3.4.1#85da54eb530cab53de0cc0ee9065d066 - Cache
    pcre2/10.42#ac2227071f79a062167cf2fee7189d66 - Cache
    qt/6.8.3#d5edf94e21dda62521720a103b1510a0 - Cache
    sqlite3/3.49.1#8631739a4c9b93bd3d6b753bac548a63 - Cache
    zlib/1.3.1#b8bc2603263cf7eccbd6e17e66b0ed76 - Cache
Build requirements
    autoconf/2.71#f9307992909d7fb3df459340f1932809 - Cache
    automake/1.16.5#058bda3e21c36c9aa8425daf3c1faf50 - Cache
    bzip2/1.0.8#d00dac990f08d991998d624be81a9526 - Cache
    cmake/3.31.6#ed0e6c1d49bd564ce6fed1a19653b86d - Cache
    glib/2.78.3#a15f775789e881a2bc3c9693aa69b2c6 - Cache
    libffi/3.4.4#1520e366dc9a6a38898b4993bb198748 - Cache
    libgettext/0.22#35d2811b2dd27a98f69e4daa86ca2000 - Cache
    libiconv/1.17#1ae2f60ab5d08de1643a22a81b360c59 - Cache
    m4/1.4.19#b38ced39a01e31fef5435bc634461fd2 - Cache
    meson/1.2.2#21b73818ba96d9eea465b310b5bbc993 - Cache
    meson/1.3.2#26ce8a76a36cc275cdfee1d757bc6561 - Cache
    meson/1.4.0#2262941cc8fbb0099dd0c196ca2a6c01 - Cache
    meson/1.6.0#df190c5b0592572e6f5a1b05999e7e6d - Cache
    msys2/cci.latest#4927aa5502eb174e95df524f2be2685e - Cache
    nasm/2.16.01#d0aebbd20ccbb6ad9c9c753ab708098c - Cache
    ninja/1.13.1#294f8721dbcde145674f7ba44994700e - Cache
    pcre2/10.42#ac2227071f79a062167cf2fee7189d66 - Cache
    pkgconf/2.1.0#27f44583701117b571307cf5b5fe5605 - Cache
    pkgconf/2.2.0#6462942a22803086372db44689ba825f - Cache
    strawberryperl/5.32.1.1#707032463aa0620fa17ec0d887f5fe41 - Cache
    zlib/1.3.1#b8bc2603263cf7eccbd6e17e66b0ed76 - Cache
Resolved version ranges
    cmake/[>=3.21.1 <4]: cmake/3.31.6
    harfbuzz/[>=8.3.0]: harfbuzz/8.3.0
    libpng/[>=1.6 <2]: libpng/1.6.47
    meson/[>=1.2.3 <2]: meson/1.6.0
    ninja/[>=1.10.2 <2]: ninja/1.13.1
    ninja/[>=1.12 <2]: ninja/1.13.1
    openssl/[>=1.1 <4]: openssl/3.4.1
    pkgconf/[>=2.2 <3]: pkgconf/2.2.0
    sqlite3/[>=3.45.0 <4]: sqlite3/3.49.1
    zlib/[>=1.2.11 <2]: zlib/1.3.1

======== Computing necessary packages ========
double-conversion/3.3.0: Main binary package 'a1a4f9c143ce4438c1ed53b4e73368879031fc25' missing
double-conversion/3.3.0: Checking 7 compatible configurations
double-conversion/3.3.0: Found compatible package '4631f2a20e70ab5c624ced8299b9b57d4e967722': compiler.cppstd=14
Requirements
    brotli/1.1.0#d56d7bb9ca722942aba17369cb5c0519:e61b328c57a7dac78df341bc28d447e08bf608e7#f7dc0afc5458c7d5e715e1eb42a3eea3 - Cache
    bzip2/1.0.8#d00dac990f08d991998d624be81a9526:dd62ce301c6ef0d8f7729c263acbaf3a11b7c621#0894c8d34c04ad66792fd72f99f77eb9 - Cache
    double-conversion/3.3.0#33321c201741cc32b51169c6d2d05e60:4631f2a20e70ab5c624ced8299b9b57d4e967722#c97d34303074175f3c2776dba6829f27 - Cache
    freetype/2.13.2#5d2563803c8558d4ef47271a82c73d20:8c8e819f72bf238a61323610becf843e3fd7289a#7f4d9b12b9aeec14097ad357a1064159 - Cache
    glib/2.78.3#a15f775789e881a2bc3c9693aa69b2c6:64e58614ad0497470a821f116bb438635379aa90#792293e50cfbd715c69d9ddbe25f1889 - Cache
    harfbuzz/8.3.0#c83405ed297bad6c9e9587e2a19c686a:2adeb6124965139576da786215b3432ead4f5f11#679b72977357cc504a755d31b15fdc1c - Cache
    libffi/3.4.4#1520e366dc9a6a38898b4993bb198748:8b7e35f91e516e42ed736c6dd52a9263da3a5dad#64cff6b1a2a648f9247ae675f4626066 - Cache
    libgettext/0.22#35d2811b2dd27a98f69e4daa86ca2000:f2a62fa0c569116b0ca2be4b88f1a2a7abecec52#2c130ea9f6cee67ae8287b0b502998fa - Cache
    libiconv/1.17#1ae2f60ab5d08de1643a22a81b360c59:8b7e35f91e516e42ed736c6dd52a9263da3a5dad#920ad1f251e9972e58cea31dacbb5333 - Cache
    libpng/1.6.47#d7ad969de6b0a72704cf03d6ef17648b:34b444759868b4ce4f04f81d4ad4927bd78d7d3e#ff39234e3a5619250ccbd3c3f80bc801 - Cache
    libpq/15.4#cca9970f349bcdb6c8b35acd017ee5aa:4b25184719d2424b6b2df1be16651e46ba3c91db#02eb5a50ddb10940c549e6660f86fbc8 - Cache
    md4c/0.4.8#d7347fea4b215dfc7041f88a539508d0:60708e6e0fba7d9b7adb9a44b2831aebd316ba0f#e8d83da0bce6c9925ff61dbcd302d0b0 - Cache
    openssl/3.4.1#85da54eb530cab53de0cc0ee9065d066:c6796611a63334e83ec731d8adba99e6f1492955#8d14ac1b2ae5d2b2c139274ad16ac25d - Cache
    pcre2/10.42#ac2227071f79a062167cf2fee7189d66:3fcf714ef0ef9d2ebeb991d58a850fb6b27c3d91#bf6bb1b2c1dc7eb2ec2b8c7d3cac947d - Cache
    qt/6.8.3#d5edf94e21dda62521720a103b1510a0:2cebb820d07e9077f6d86f467f56eb9fde326294#2d4943143fa921c7d95a0ff7a354c67a - Cache
    sqlite3/3.49.1#8631739a4c9b93bd3d6b753bac548a63:27478b00f24e5839eaa4503c51b7af328e225705#eb7dda79282ab684b08e5b7ce177a009 - Cache
    zlib/1.3.1#b8bc2603263cf7eccbd6e17e66b0ed76:8b7e35f91e516e42ed736c6dd52a9263da3a5dad#3bba0747d430802012963ec1c021b5d4 - Cache
Build requirements
Skipped binaries
    autoconf/2.71, automake/1.16.5, bzip2/1.0.8, cmake/3.31.6, glib/2.78.3, libffi/3.4.4, libgettext/0.22, libiconv/1.17, m4/1.4.19, meson/1.2.2, meson/1.3.2, meson/1.4.0, meson/1.6.0, msys2/cci.latest, nasm/2.16.01, ninja/1.13.1, pcre2/10.42, pkgconf/2.1.0, pkgconf/2.2.0, strawberryperl/5.32.1.1, zlib/1.3.1

======== Installing packages ========
brotli/1.1.0: Already installed! (1 of 17)
bzip2/1.0.8: Already installed! (2 of 17)
double-conversion/3.3.0: Already installed! (3 of 17)
md4c/0.4.8: Already installed! (4 of 17)
sqlite3/3.49.1: Already installed! (5 of 17)
zlib/1.3.1: Already installed! (6 of 17)
libiconv/1.17: Already installed! (7 of 17)
libpq/15.4: Already installed! (8 of 17)
libffi/3.4.4: Already installed! (9 of 17)
libpng/1.6.47: Already installed! (10 of 17)
pcre2/10.42: Already installed! (11 of 17)
pcre2/10.42: Appending PATH environment variable: C:\Users\usernameredacted\.conan2_yael\p\pcre2d127935f7dd40\p\bin
openssl/3.4.1: Already installed! (12 of 17)
libgettext/0.22: Already installed! (13 of 17)
freetype/2.13.2: Already installed! (14 of 17)
glib/2.78.3: Already installed! (15 of 17)
harfbuzz/8.3.0: Already installed! (16 of 17)
qt/6.8.3: Already installed! (17 of 17)
WARN: deprecated: Usage of deprecated Conan 1.X features that will be removed in Conan 2.X:
WARN: deprecated:     'cpp_info.names' used in: freetype/2.13.2, bzip2/1.0.8, pcre2/10.42, brotli/1.1.0, libpng/1.6.47, libpq/15.4, zlib/1.3.1, libgettext/0.22
WARN: deprecated:     'cpp_info.build_modules' used in: freetype/2.13.2, bzip2/1.0.8
WARN: deprecated:     'env_info' used in: freetype/2.13.2, bzip2/1.0.8, libpq/15.4, pcre2/10.42, glib/2.78.3
WARN: deprecated:     'cpp_info.filenames' used in: freetype/2.13.2
WARN: deprecated:     'user_info' used in: freetype/2.13.2

======== Testing the package ========
Removing previously existing 'test_package' build folder: C:\Users\usernameredacted\Projets\conan-center-index\recipes\qt\6.x.x\test_package\build\msvc-194-x86_64-17-debug
qt/6.8.3 (test package): Test package build: build\msvc-194-x86_64-17-debug
qt/6.8.3 (test package): Test package build folder: C:\Users\usernameredacted\Projets\conan-center-index\recipes\qt\6.x.x\test_package\build\msvc-194-x86_64-17-debug
qt/6.8.3 (test package): Writing generators to C:\Users\usernameredacted\Projets\conan-center-index\recipes\qt\6.x.x\test_package\build\msvc-194-x86_64-17-debug\generators
qt/6.8.3 (test package): Generator 'CMakeDeps' calling 'generate()'
qt/6.8.3 (test package): CMakeDeps necessary find_package() and targets for your CMakeLists.txt
    find_package(Qt6)
    target_link_libraries(... qt::qt)
qt/6.8.3 (test package): Generator 'VirtualBuildEnv' calling 'generate()'
qt/6.8.3 (test package): Calling generate()
qt/6.8.3 (test package): Generators folder: C:\Users\usernameredacted\Projets\conan-center-index\recipes\qt\6.x.x\test_package\build\msvc-194-x86_64-17-debug\generators
qt/6.8.3 (test package): CMakeToolchain generated: conan_toolchain.cmake
qt/6.8.3 (test package): CMakeToolchain generated: C:\Users\usernameredacted\Projets\conan-center-index\recipes\qt\6.x.x\test_package\build\msvc-194-x86_64-17-debug\generators\CMakePresets.json
qt/6.8.3 (test package): CMakeToolchain generated: C:\Users\usernameredacted\Projets\conan-center-index\recipes\qt\6.x.x\test_package\CMakeUserPresets.json
qt/6.8.3 (test package): Generating aggregated env files
qt/6.8.3 (test package): Generated aggregated env files: ['conanbuild.bat', 'conanrun.bat']

======== Testing the package: Building ========
qt/6.8.3 (test package): Calling build()
qt/6.8.3 (test package): Running CMake.configure()
qt/6.8.3 (test package): RUN: cmake -G "Ninja" -DCMAKE_TOOLCHAIN_FILE="generators/conan_toolchain.cmake" -DCMAKE_INSTALL_PREFIX="C:/Users/usernameredacted/Projets/conan-center-index/recipes/qt/6.x.x/test_package" -DCMAKE_POLICY_DEFAULT_CMP0091="NEW" -DCMAKE_BUILD_TYPE="Debug" "C:/Users/usernameredacted/Projets/conan-center-index/recipes/qt/6.x.x/test_package"
conanvcvars.bat: Activating environment Visual Studio 17 - amd64 - winsdk_version=None - vcvars_ver=14.4
[vcvarsall.bat] Environment initialized for: 'x64'
-- Using Conan toolchain: C:/Users/usernameredacted/Projets/conan-center-index/recipes/qt/6.x.x/test_package/build/msvc-194-x86_64-17-debug/generators/conan_toolchain.cmake
-- Conan toolchain: Setting CMAKE_MSVC_RUNTIME_LIBRARY=$<$<CONFIG:Debug>:MultiThreadedDebugDLL>
-- Conan toolchain: C++ Standard 17 with extensions OFF
-- The CXX compiler identification is MSVC 19.43.34809.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.43.34808/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Conan: Component target declared 'Qt6::Platform'
-- Conan: Component target declared 'qt::QODBCDriverPlugin'
-- Conan: Component target declared 'Qt6::LinguistTools'
-- Conan: Component target declared 'Qt6::EntryPointImplementation'
-- Conan: Component target declared 'Qt6::Core'
-- Conan: Component target declared 'Qt6::EntryPointPrivate'
-- Conan: Component target declared 'Qt6::Gui'
-- Conan: Component target declared 'Qt6::QSQLiteDriverPlugin'
-- Conan: Component target declared 'Qt6::QPSQLDriverPlugin'
-- Conan: Component target declared 'Qt6::QODBCDriverPlugin'
-- Conan: Component target declared 'Qt6::Network'
-- Conan: Component target declared 'Qt6::Sql'
-- Conan: Component target declared 'Qt6::Test'
-- Conan: Component target declared 'Qt6::Concurrent'
-- Conan: Component target declared 'Qt6::Xml'
-- Conan: Component target declared 'Qt6::QSvgIconPlugin'
-- Conan: Component target declared 'Qt6::QSvgPlugin'
-- Conan: Component target declared 'Qt6::QWindowsIntegrationPlugin'
-- Conan: Component target declared 'Qt6::QModernWindowsStylePlugin'
-- Conan: Component target declared 'Qt6::QGifPlugin'
-- Conan: Component target declared 'Qt6::QIcoPlugin'
-- Conan: Component target declared 'Qt6::Widgets'
-- Conan: Component target declared 'Qt6::OpenGL'
-- Conan: Component target declared 'Qt6::ShaderTools'
-- Conan: Component target declared 'Qt6::Svg'
-- Conan: Component target declared 'Qt6::Multimedia'
-- Conan: Component target declared 'Qt6::PrintSupport'
-- Conan: Component target declared 'Qt6::OpenGLWidgets'
-- Conan: Component target declared 'Qt6::UiPlugin'
-- Conan: Component target declared 'Qt6::Help'
-- Conan: Component target declared 'Qt6::SvgWidgets'
-- Conan: Component target declared 'Qt6::MultimediaWidgets'
-- Conan: Component target declared 'Qt6::UiTools'
-- Conan: Component target declared 'Qt6::Designer'
-- Conan: Target declared 'qt::qt'
-- Conan: Component target declared 'OpenSSL::Crypto'
-- Conan: Component target declared 'OpenSSL::SSL'
-- Conan: Target declared 'openssl::openssl'
-- Conan: Target declared 'ZLIB::ZLIB'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/opensbfd10520b848f/p/lib/cmake/conan-official-openssl-variables.cmake'
-- Conan: Target declared 'double-conversion::double-conversion'
-- Conan: Target declared 'harfbuzz::harfbuzz'
-- Conan: Target declared 'Freetype::Freetype'
-- Conan: Target declared 'BZip2::BZip2'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/bzip24357523925bd0/p/lib/cmake/conan-official-bzip2-variables.cmake'
-- Conan: Target declared 'PNG::PNG'
-- Conan: Component target declared 'brotli::brotlicommon'
-- Conan: Component target declared 'brotli::brotlidec'
-- Conan: Component target declared 'brotli::brotlienc'
-- Conan: Target declared 'brotli::brotli'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/freet17d05e5ec1f56/p/lib/cmake/conan-official-freetype-variables.cmake'
-- Conan: Component target declared 'glib::glib-2.0'
-- Conan: Component target declared 'glib::gresource'
-- Conan: Component target declared 'glib::gmodule-no-export-2.0'
-- Conan: Component target declared 'glib::gobject-2.0'
-- Conan: Component target declared 'glib::gthread-2.0'
-- Conan: Component target declared 'glib::gmodule-export-2.0'
-- Conan: Component target declared 'glib::gmodule-2.0'
-- Conan: Component target declared 'glib::gio-2.0'
-- Conan: Component target declared 'glib::gio-windows-2.0'
-- Conan: Target declared 'glib::glib'
-- Conan: Target declared 'libffi::libffi'
-- Conan: Component target declared 'PCRE2::8BIT'
-- Conan: Component target declared 'PCRE2::16BIT'
-- Conan: Component target declared 'PCRE2::32BIT'
-- Conan: Component target declared 'PCRE2::POSIX'
-- Conan: Target declared 'pcre2::pcre2'
-- Conan: Target declared 'Intl::Intl'
-- Conan: Target declared 'Iconv::Iconv'
-- Conan: Component target declared 'SQLite::SQLite3'
-- Conan: Component target declared 'libpq::pgport'
-- Conan: Component target declared 'libpq::pgcommon'
-- Conan: Component target declared 'libpq::pq'
-- Conan: Target declared 'PostgreSQL::PostgreSQL'
-- Conan: Component target declared 'md4c::md4c'
-- Conan: Component target declared 'md4c::md4c-html'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/conan_qt_platform_target_setup.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6Core/conan_qt_executables_variables.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6Core/conan_qt_qt6_coreprivate.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6Core/conan_qt_entry_point.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/Qt6ConfigExtras.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicAppleHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicCMakeHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicCMakeVersionHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicDependencyHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicExternalProjectHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicFinalizerHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicFindPackageHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicGitHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicPluginHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicSbomAttributionHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicSbomCpeHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicSbomDepHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicSbomFileHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicSbomGenerationHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicSbomHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicSbomLicenseHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicSbomOpsHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicSbomPurlHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicSbomPythonHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicSbomQtEntityHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicSbomSystemDepHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicTargetHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicTestHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicToolHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicWalkLibsHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6/QtPublicWasmToolchainHelpers.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6Core/Qt6CoreMacros.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6Core/Qt6CoreConfigExtras.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6Gui/conan_qt_qt6_guiprivate.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6Widgets/conan_qt_qt6_widgetsprivate.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6Widgets/Qt6WidgetsMacros.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6LinguistTools/Qt6LinguistToolsMacros.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6ShaderToolsTools/Qt6ShaderToolsMacros.cmake'
-- Conan: Including build module from 'C:/Users/usernameredacted/.conan2_yael/p/b/qt3c0a0868af597/p/lib/cmake/Qt6Multimedia/Qt6MultimediaMacros.cmake'
-- Configuring done (1.6s)
-- Generating done (0.4s)
-- Build files have been written to: C:/Users/usernameredacted/Projets/conan-center-index/recipes/qt/6.x.x/test_package/build/msvc-194-x86_64-17-debug

qt/6.8.3 (test package): Running CMake.build()
qt/6.8.3 (test package): RUN: cmake --build "C:\Users\usernameredacted\Projets\conan-center-index\recipes\qt\6.x.x\test_package\build\msvc-194-x86_64-17-debug" -- -j16
conanvcvars.bat: Activating environment Visual Studio 17 - amd64 - winsdk_version=None - vcvars_ver=14.4
[vcvarsall.bat] Environment initialized for: 'x64'
[6/6] Linking CXX executable test_package.exe
FAILED: [code=4294967295] test_package.exe
C:\WINDOWS\system32\cmd.exe /C "cd . && C:\Users\usernameredacted\Portable\cmake-3.30.4-windows-x86_64\bin\cmake.exe -E vs_link_exe --intdir=CMakeFiles\test_package.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MIB055~1\2022\PROFES~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\link.exe /nologo @CMakeFiles\test_package.rsp  /out:test_package.exe /implib:test_package.lib /pdb:test_package.pdb /version:0.0 /machine:x64 /debug /INCREMENTAL /subsystem:console  C:\Users\usernameredacted\.conan2_yael\p\b\qt3c0a0868af597\p\lib\objects-Debug\Widgets_resources_1\.qt\rcc\qrc_qstyle_init.cpp.obj C:\Users\usernameredacted\.conan2_yael\p\b\qt3c0a0868af597\p\lib\objects-Debug\Widgets_resources_2\.qt\rcc\qrc_qstyle1_init.cpp.obj C:\Users\usernameredacted\.conan2_yael\p\b\qt3c0a0868af597\p\lib\objects-Debug\Widgets_resources_3\.qt\rcc\qrc_qstyle_fusion_init.cpp.obj C:\Users\usernameredacted\.conan2_yael\p\b\qt3c0a0868af597\p\lib\objects-Debug\Widgets_resources_4\.qt\rcc\qrc_qmessagebox_init.cpp.obj C:\Users\usernameredacted\.conan2_yael\p\b\qt3c0a0868af597\p\lib\objects-Debug\Gui_resources_1\.qt\rcc\qrc_qpdf_init.cpp.obj C:\Users\usernameredacted\.conan2_yael\p\b\qt3c0a0868af597\p\lib\objects-Debug\Gui_resources_2\.qt\rcc\qrc_gui_shaders_init.cpp.obj C:\Users\usernameredacted\.conan2_yael\p\b\qt3c0a0868af597\p\lib\objects-Debug\QWindowsIntegrationPlugin_resources_1\.qt\rcc\qrc_openglblacklists_init.cpp.obj C:\Users\usernameredacted\.conan2_yael\p\b\qt3c0a0868af597\p\lib\objects-Debug\QWindowsIntegrationPlugin_resources_2\.qt\rcc\qrc_cursors_init.cpp.obj && cd ."
LINK Pass 1: command "C:\PROGRA~1\MIB055~1\2022\PROFES~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\link.exe /nologo @CMakeFiles\test_package.rsp /out:test_package.exe /implib:test_package.lib /pdb:test_package.pdb /version:0.0 /machine:x64 /debug /INCREMENTAL /subsystem:console C:\Users\usernameredacted\.conan2_yael\p\b\qt3c0a0868af597\p\lib\objects-Debug\Widgets_resources_1\.qt\rcc\qrc_qstyle_init.cpp.obj C:\Users\usernameredacted\.conan2_yael\p\b\qt3c0a0868af597\p\lib\objects-Debug\Widgets_resources_2\.qt\rcc\qrc_qstyle1_init.cpp.obj C:\Users\usernameredacted\.conan2_yael\p\b\qt3c0a0868af597\p\lib\objects-Debug\Widgets_resources_3\.qt\rcc\qrc_qstyle_fusion_init.cpp.obj C:\Users\usernameredacted\.conan2_yael\p\b\qt3c0a0868af597\p\lib\objects-Debug\Widgets_resources_4\.qt\rcc\qrc_qmessagebox_init.cpp.obj C:\Users\usernameredacted\.conan2_yael\p\b\qt3c0a0868af597\p\lib\objects-Debug\Gui_resources_1\.qt\rcc\qrc_qpdf_init.cpp.obj C:\Users\usernameredacted\.conan2_yael\p\b\qt3c0a0868af597\p\lib\objects-Debug\Gui_resources_2\.qt\rcc\qrc_gui_shaders_init.cpp.obj C:\Users\usernameredacted\.conan2_yael\p\b\qt3c0a0868af597\p\lib\objects-Debug\QWindowsIntegrationPlugin_resources_1\.qt\rcc\qrc_openglblacklists_init.cpp.obj C:\Users\usernameredacted\.conan2_yael\p\b\qt3c0a0868af597\p\lib\objects-Debug\QWindowsIntegrationPlugin_resources_2\.qt\rcc\qrc_cursors_init.cpp.obj /MANIFEST /MANIFESTFILE:CMakeFiles\test_package.dir/intermediate.manifest CMakeFiles\test_package.dir/manifest.res" failed (exit code 1120) with the following output:
double-conversion.lib(string-to-double.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'double-conversion.lib(string-to-double.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
double-conversion.lib(double-to-string.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'double-conversion.lib(double-to-string.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
double-conversion.lib(strtod.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'double-conversion.lib(strtod.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
double-conversion.lib(bignum-dtoa.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'double-conversion.lib(bignum-dtoa.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
double-conversion.lib(fast-dtoa.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'double-conversion.lib(fast-dtoa.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
double-conversion.lib(fixed-dtoa.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'double-conversion.lib(fixed-dtoa.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
double-conversion.lib(bignum.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'double-conversion.lib(bignum.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
double-conversion.lib(cached-powers.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'double-conversion.lib(cached-powers.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-common.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-common.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-buffer.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-buffer.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-shape.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-shape.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-tag.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-tag.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-layout.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-layout.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-blob.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-blob.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-unicode.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-unicode.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-face.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-face.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-font.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-font.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-number.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-number.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-shape-plan.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-shape-plan.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-shaper.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-shaper.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-buffer-verify.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-buffer-verify.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-set.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-set.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-metrics.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-metrics.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-aat-layout.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-aat-layout.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ucd.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ucd.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-face.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-face.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-shape.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-shape.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-uniscribe.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-uniscribe.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-fallback-shape.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-fallback-shape.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-font.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-font.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-var.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-var.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-buffer-serialize.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-buffer-serialize.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-aat-map.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-aat-map.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-map.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-map.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-shape-normalize.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-shape-normalize.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-shape-fallback.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-shape-fallback.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-shaper-arabic.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-shaper-arabic.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-shaper-default.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-shaper-default.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-shaper-hangul.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-shaper-hangul.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-shaper-hebrew.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-shaper-hebrew.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-shaper-indic.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-shaper-indic.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-shaper-khmer.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-shaper-khmer.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-shaper-myanmar.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-shaper-myanmar.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-shaper-thai.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-shaper-thai.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-shaper-use.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-shaper-use.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-color.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-color.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-outline.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-outline.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-cff2-table.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-cff2-table.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-cff1-table.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-cff1-table.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-paint-extents.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-paint-extents.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-shaper-syllabic.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-shaper-syllabic.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-shaper-indic-table.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-shaper-indic-table.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-ot-shaper-vowel-constraints.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-ot-shaper-vowel-constraints.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-draw.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-draw.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
harfbuzz.lib(hb-paint.cc.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'harfbuzz.lib(hb-paint.cc.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_base_ftbase.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_base_ftbase.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_base_ftmm.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_base_ftmm.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_base_ftinit.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_base_ftinit.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_base_fttype1.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_base_fttype1.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_base_ftsynth.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_base_ftsynth.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(builds_windows_ftsystem.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(builds_windows_ftsystem.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_truetype_truetype.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_truetype_truetype.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_type1_type1.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_type1_type1.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_cff_cff.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_cff_cff.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_cid_type1cid.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_cid_type1cid.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_pfr_pfr.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_pfr_pfr.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_type42_type42.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_type42_type42.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_winfonts_winfnt.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_winfonts_winfnt.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_pcf_pcf.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_pcf_pcf.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_bdf_bdf.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_bdf_bdf.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_sfnt_sfnt.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_sfnt_sfnt.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_autofit_autofit.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_autofit_autofit.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_pshinter_pshinter.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_pshinter_pshinter.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_smooth_smooth.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_smooth_smooth.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_raster_raster.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_raster_raster.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_svg_svg.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_svg_svg.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_sdf_sdf.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_sdf_sdf.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_psaux_psaux.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_psaux_psaux.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_psnames_psnames.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_psnames_psnames.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_base_ftbitmap.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_base_ftbitmap.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_gzip_ftgzip.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_gzip_ftgzip.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_lzw_ftlzw.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_lzw_ftlzw.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(src_bzip2_ftbzip2.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(src_bzip2_ftbzip2.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
freetype.lib(builds_windows_ftdebug.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'freetype.lib(builds_windows_ftdebug.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
brotlidec.lib(decode.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'brotlidec.lib(decode.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
brotlidec.lib(bit_reader.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'brotlidec.lib(bit_reader.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
brotlidec.lib(huffman.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'brotlidec.lib(huffman.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
brotlidec.lib(state.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'brotlidec.lib(state.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
brotlicommon.lib(shared_dictionary.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'brotlicommon.lib(shared_dictionary.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
brotlicommon.lib(transform.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'brotlicommon.lib(transform.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
brotlicommon.lib(constants.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'brotlicommon.lib(constants.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
brotlicommon.lib(context.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'brotlicommon.lib(context.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
brotlicommon.lib(platform.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'brotlicommon.lib(platform.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
brotlicommon.lib(dictionary.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'brotlicommon.lib(dictionary.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
libpng16_staticd.lib(png.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'libpng16_staticd.lib(png.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
libpng16_staticd.lib(pngread.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'libpng16_staticd.lib(pngread.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
libpng16_staticd.lib(pngwrite.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'libpng16_staticd.lib(pngwrite.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
libpng16_staticd.lib(pngerror.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'libpng16_staticd.lib(pngerror.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
libpng16_staticd.lib(pngwutil.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'libpng16_staticd.lib(pngwutil.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
libpng16_staticd.lib(pngrtran.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'libpng16_staticd.lib(pngrtran.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
libpng16_staticd.lib(pngtrans.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'libpng16_staticd.lib(pngtrans.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
libpng16_staticd.lib(pngwio.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'libpng16_staticd.lib(pngwio.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
libpng16_staticd.lib(pngrio.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'libpng16_staticd.lib(pngrio.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
libpng16_staticd.lib(pngset.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'libpng16_staticd.lib(pngset.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
libpng16_staticd.lib(pngget.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'libpng16_staticd.lib(pngget.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
libpng16_staticd.lib(pngmem.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'libpng16_staticd.lib(pngmem.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
libpng16_staticd.lib(pngrutil.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'libpng16_staticd.lib(pngrutil.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
libpng16_staticd.lib(pngwtran.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'libpng16_staticd.lib(pngwtran.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
libpng16_staticd.lib(intel_init.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'libpng16_staticd.lib(intel_init.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
libpng16_staticd.lib(filter_sse2_intrinsics.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'libpng16_staticd.lib(filter_sse2_intrinsics.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_config.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_config.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_context.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_context.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_compile.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_compile.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_pattern_info.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_pattern_info.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_match_data.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_match_data.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_match.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_match.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_jit_compile.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_jit_compile.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_error.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_error.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_string_utils.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_string_utils.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_ucd.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_ucd.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_chartables.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_chartables.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_auto_possess.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_auto_possess.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_find_bracket.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_find_bracket.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_newline.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_newline.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_ord2utf.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_ord2utf.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_study.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_study.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_valid_utf.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_valid_utf.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_tables.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_tables.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_extuni.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_extuni.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_script_run.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_script_run.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
pcre2-16-staticd.lib(pcre2_xclass.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'pcre2-16-staticd.lib(pcre2_xclass.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
zlib.lib(inflate.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'zlib.lib(inflate.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
zlib.lib(deflate.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'zlib.lib(deflate.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
zlib.lib(compress.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'zlib.lib(compress.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
zlib.lib(uncompr.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'zlib.lib(uncompr.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
zlib.lib(crc32.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'zlib.lib(crc32.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
zlib.lib(adler32.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'zlib.lib(adler32.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
zlib.lib(zutil.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'zlib.lib(zutil.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
zlib.lib(inftrees.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'zlib.lib(inftrees.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
zlib.lib(inffast.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'zlib.lib(inffast.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
zlib.lib(trees.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'zlib.lib(trees.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
bz2.lib(bzlib.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'bz2.lib(bzlib.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
bz2.lib(compress.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'bz2.lib(compress.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
bz2.lib(decompress.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'bz2.lib(decompress.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
bz2.lib(randtable.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'bz2.lib(randtable.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
bz2.lib(crctable.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'bz2.lib(crctable.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
bz2.lib(blocksort.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'bz2.lib(blocksort.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
bz2.lib(huffman.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'bz2.lib(huffman.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
md4c.lib(md4c.c.obj) : warning LNK4099: PDB '' n'a pu être trouvé avec 'md4c.lib(md4c.c.obj)' ou sur '' ; l'objet sera lié sans informations de débogage
qwindowsd.lib(qwindowsuiaaccessibility.cpp.obj) : error LNK2019: symbole externe non résolu UiaClientsAreListening référencé dans la fonction "public: virtual void __cdecl QWindowsUiaAccessibility::notifyAccessibilityUpdate(class QAccessibleEvent *)" (?notifyAccessibilityUpdate@QWindowsUiaAccessibility@@UEAAXPEAVQAccessibleEvent@@@Z)
qwindowsd.lib(qwindowsuiaaccessibility.cpp.obj) : error LNK2019: symbole externe non résolu UiaReturnRawElementProvider référencé dans la fonction "public: static bool __cdecl QWindowsUiaAccessibility::handleWmGetObject(struct HWND__ *,unsigned __int64,__int64,__int64 *)" (?handleWmGetObject@QWindowsUiaAccessibility@@SA_NPEAUHWND__@@_K_JPEA_J@Z)
qwindowsd.lib(qwindowsuiamainprovider.cpp.obj) : error LNK2019: symbole externe non résolu UiaRaiseAutomationPropertyChangedEvent référencé dans la fonction "public: static void __cdecl QWindowsUiaMainProvider::notifyStateChange(class QAccessibleStateChangeEvent *)" (?notifyStateChange@QWindowsUiaMainProvider@@SAXPEAVQAccessibleStateChangeEvent@@@Z)
qwindowsd.lib(qwindowsuiamainprovider.cpp.obj) : error LNK2019: symbole externe non résolu UiaRaiseAutomationEvent référencé dans la fonction "public: static void __cdecl QWindowsUiaMainProvider::notifyFocusChange(class QAccessibleEvent *)" (?notifyFocusChange@QWindowsUiaMainProvider@@SAXPEAVQAccessibleEvent@@@Z)
qwindowsd.lib(qwindowsuiamainprovider.cpp.obj) : error LNK2019: symbole externe non résolu UiaRaiseNotificationEvent référencé dans la fonction "public: static void __cdecl QWindowsUiaMainProvider::raiseNotification(class QAccessibleAnnouncementEvent *)" (?raiseNotification@QWindowsUiaMainProvider@@SAXPEAVQAccessibleAnnouncementEvent@@@Z)
qwindowsd.lib(qwindowsuiamainprovider.cpp.obj) : error LNK2019: symbole externe non résolu UiaHostProviderFromHwnd référencé dans la fonction "public: virtual long __cdecl QWindowsUiaMainProvider::get_HostRawElementProvider(struct IRawElementProviderSimple * *)" (?get_HostRawElementProvider@QWindowsUiaMainProvider@@UEAAJPEAPEAUIRawElementProviderSimple@@@Z)
test_package.exe : fatal error LNK1120: 6 externes non résolus
ninja: build stopped: subcommand failed.

ERROR: qt/6.8.3 (test package): Error in build() method, line 45
        cmake.build()
        ConanException: Error 4294967295 while executing
```

</details>

--- a/recipes/qt/6.x.x/conanfile.py
+++ b/recipes/qt/6.x.x/conanfile.py
@@ -1177,8 +1177,6 @@ class QtConan(ConanFile):
                 if Version(self.version) >= "6.7.0":
                     # https://github.com/qt/qtbase/blob/v6.7.0-beta1/src/gui/CMakeLists.txt#L430
                     self.cpp_info.components["qtGui"].system_libs.append("uxtheme")
-                if self.settings.compiler == "gcc":
-                    self.cpp_info.components["qtGui"].system_libs.append("uuid")
                 # https://github.com/qt/qtbase/blob/v6.6.1/src/plugins/platforms/direct2d/CMakeLists.txt#L60-L82
                 self.cpp_info.components["qtGui"].system_libs += [
                     "advapi32", "d2d1", "d3d11", "dwmapi", "dwrite", "dxgi", "dxguid", "gdi32", "imm32", "ole32",
@@ -1196,6 +1194,9 @@ class QtConan(ConanFile):
                     "advapi32", "dwmapi", "gdi32", "imm32", "ole32", "oleaut32", "setupapi", "shell32", "shlwapi",
                     "user32", "winmm", "winspool", "wtsapi32", "shcore", "comdlg32", "d3d9", "runtimeobject"
                 ]
+                if Version(self.version) >= "6.8.0" and is_msvc(self):
+                    # https://github.com/qt/qtbase/blob/6.8.3/src/plugins/platforms/windows/CMakeLists.txt#L204
+                    self.cpp_info.components["qtQWindowsIntegrationPlugin"].system_libs.append("uiautomationcore")
             elif self.settings.os == "Android":
                 _create_plugin("QAndroidIntegrationPlugin", "qtforandroid", "platforms", ["Core", "Gui"])
                 # https://github.com/qt/qtbase/blob/v6.6.1/src/plugins/platforms/android/CMakeLists.txt#L68-L70

